src/main/java/org/hobbit/utils/kubernetes/KubeHelper.java

```diff
@@ -0,0 +1,40 @@
+package org.hobbit.utils.kubernetes;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+public class KubeHelper {
+    private static final Logger LOGGER = LoggerFactory.getLogger(KubeHelper.class);
+    /**
+     * Gets dns friendly ip
```

Comment:
"Get network identifier that can be used to reach this container. This can be either a DNS name, an IP address or something similar."

Reply:
here for Kubernetes it would be IP